controllers/user.js

```diff
@@ -26,45 +26,45 @@ module.exports = {
       }).then(ticket => {
         const payload = ticket.getPayload();
 
-        doesSuchSellerExist(payload.email)
+        doesSuchUserExist(payload.email)
           .then(result => {
             if (result) {
               result = result.dataValues;
               return resolve([
                 {
                   id: result.id,
                   mobile: result.mobile,
-                  name: result.name,
+                  firstName: result.firstName,
```

Comment:
aren't we missing lastname?

Reply:
```
id, mobile, firstName, email, and ProfilePic
``` are mandatory. (`mobile`) can be omitted but other params are critical. This info is not profile details but login details. We are returning all params in /users/:id get.

Reply:
Ok.